tests/operators/test_python.py

```diff
@@ -617,7 +617,7 @@ def test_xcom_push(self):
 
         tis = dr.get_task_instances()
         assert tis[0].xcom_pull(task_ids=short_op_push_xcom.task_id, key="return_value") == "signature"
-        assert tis[0].xcom_pull(task_ids=short_op_no_push_xcom.task_id, key="return_value") is None
+        assert tis[0].xcom_pull(task_ids=short_op_no_push_xcom.task_id, key="return_value") is False
```

Comment:
Is this considered a breaking change? If yes, how do you think we can handle it?